swagger.yaml

```diff
@@ -2867,6 +2867,13 @@ definitions:
         type: string
         description: OpenShift cluster name.
         x-nullable: true
+      mode:
```

Comment:
What other enum values do we envision in the future?
Either we should change "mode" to something more specific like "resiliency", and in the future, we can add active-passive, or maybe we should leave it as "mode" if we want to add other non-resiliency values.  I'm guessing the first option.

Reply:
This should match with https://github.com/openshift/api/pull/827/files#diff-3ff94e87467e8743ff79a40fa85f123d3205a560f79b4dea1919e76a4d55958dR181

Reply:
done